docs/baking.md

```diff
@@ -52,35 +59,37 @@ sudo add-apt-repository 'deb http://ppa.launchpad.net/serokell/tezos/ubuntu bion
 sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 37B8819B7D0D183812DCA9A8CE5A4D8933AE7CBB
 sudo apt-get update
 # Install packages
-sudo apt-get install tezos-client tezos-node tezos-baker-<proto> tezos-endorser-<proto>
+sudo apt-get install tezos-baking
 ```
 
 Packages for `tezos-node`, `tezos-baker-<proto>` and `tezos-endorser-<proto>` provide
-systemd units for running the corresponding binaries in the background.
-
-## Setting up the tezos-node
-
-The easiest way to set up a node running on the `mainnet` or one of the
-testnets (e.g. `edo2net`) is to use one of the predefined
-`tezos-node-<network>.services` systemd services provided in the `tezos-node`
-package.
+systemd units for running the corresponding binaries in the background, these units
+are orchestrated by the `tezos-baking-<network>` units.
 
-However, by default, these services will start to bootstrap the node from scratch,
-which will take a significant amount of time.
-In order to avoid this, we suggest bootstrapping from a snapshot instead.
+## Setting up baking service
 
-### Setting up node service
+By default `tezos-baking-<network>.service` will be using:
+* `/var/lib/.tezos-client` as the `tezos-client` data directory
+* `/var/lib/tezos/node-<network>` as the `tezos-node` data directory
+* `http://localhost:8732` as the `tezos-node` RPC address.
 
-`tezos-node-<network>.service` has `/var/lib/tezos/node-<network>` as a data directory
-and `http://localhost:8732` as its RPC address by default.
+In case you want to use a different `tezos-client` data directory or RPC address,
```

Comment:
Similar to customizing baker key alias, I think this should go to the end of the document, into "Advanced" section (which should probably also elaborate on why users may want to change this - e.g. not having adequate storage in /var). Moving it to the end will get majority of users to a working setup quicker.